Diary-Sample/Controllers/ApiController.cs

```diff
@@ -33,5 +33,9 @@ public ApiController(ILogger<ApiController> logger, IApiService service)
         [HttpGet]
         [ProducesResponseType(StatusCodes.Status200OK)]
         public ActionResult<int> Counts() => Ok(_service.Counts());
+
+        [HttpGet]
```

Comment:
リクエストパラメータでIDを指定するのでなく、パスにIDを指定する形のほうが良いと思いました。
```suggestion
        [HttpGet{id}]
```

Reply:
確かにそうですよね。忘れておりました…。
ということで、{id}を付けてみました。